README.md

```diff
@@ -47,7 +55,7 @@ Execute the commands below to build from source.
 
 As an open source project, Ballerina welcomes contributions from the community. 
 
-You can also check for [open issues](https://github.com/ballerina-platform/module-ballerina-system/issues) that interest you. We look forward to receiving your contributions.
+You can also check for [open issues](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fsystem) that interest you. We look forward to receiving your contributions.
```

Comment:
According to ballerina-platform/module-ballerina-http#99 this line has to be removed.

Reply:
Removed in a013aaa